applications/nlp/RoBERTa/roberta.py

```diff
@@ -161,6 +162,22 @@ def forward(
         embeddings = lbann.Dropout(embeddings, keep_prob=self.hidden_dropout_prob)
         return embeddings
 
+    def create_position_ids_from_inputs_embeds(self, input_embeds):
+        sequence_length = self.input_shape[1]
+        position_ids = range(
+            self.padding_idx + 1, sequence_length + self.padding_idx + 1
+        )
+        position_ids = lbann.WeightsLayer(
+            weights=lbann.Weights(
+                initializer=lbann.ValueInitializer(values=str_list(position_ids))
+            ),
+            dims=str_list([sequence_length]),
+        )
+        position_ids = lbann.Tessellate(
+            position_ids, dims=str_list(self.input_shape[:-1])
+        )
```

Comment:
Why do we need to tessellate to match the embedding vector size? If we are throwing these indices into an embedding layer, shouldn't that bring us up to the right shape?